Examples/Io/Root/include/ActsExamples/Io/Root/RootVertexPerformanceWriter.hpp

```diff
@@ -125,6 +128,26 @@ class RootVertexPerformanceWriter final
       const SimParticleContainer& collection) const;
 
   int getNumberOfTruePriVertices(const SimParticleContainer& collection) const;
+
+  ReadDataHandle<SimParticleContainer> m_inputAllTruthParticles{
+      this, "InputAllTruthParticles"};
+
+  ReadDataHandle<SimParticleContainer> m_inputSelectedTruthParticles{
+      this, "InputSelectedTruthParticles"};
+
+  ReadDataHandle<std::vector<Acts::BoundTrackParameters>>
+      m_inputTrackParameters{this, "InputTrackParameters"};
```

Comment:
would it be possible to make this handle optional and decide in the constructor if it should be initialized or not?

i think it would be nice to carry the optional inputs/outputs through like this so they don't even end up in the listed ones for the sequencer element

Reply:
That's exactly how it's implemented is actually.